src/pages/fr/guides/deploy.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Déployer votre Site Astro
+description: Comment déployer votre site Astro sur le web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiement, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
+
+<DeployGuidesNav />
+
+## Options de déploiement rapide
+
+Vous pouvez construire et déployer rapidement un site Astro vers de nombreux hébergeurs, via l'interface de leur site internet ou une CLI.
+
+### Interface d'un hébergeur
+
+Une façon rapide de déployer votre site internet est de connecter le dépôt Git (ex. GitHub, GitLab, Bitbucket) de votre projet Astro à un hébergeur, afin de profiter du déploiement continu en utilisant Git.
+
+Ces hébergeurs détectent automatiquement les push sur le dépôt de votre projet Astro, build votre site et le déploient sur le web sur une URL personnalisée ou sur votre nom de domaine personnel. La plupart du temps, mettre en place le déploiement sur ces plateformes suivra les étapes suivantes :
+
+1. Ajoutez votre dépôt à un provider Git (ex. sur GitHub, GitLab, Bitbucket)
+
+1. Choisissez un hébergeur qui supporte le *déploiement continu* (ex. [Netlify](/en/guides/deploy/netlify/) ou [Vercel](/en/guides/deploy/vercel/)) et importez votre dépôt Git en tant que nouveau projet / site.
```

Comment:
We always use the `/fr/` links to make sure we don’t pull a reader over to the English docs site.

```suggestion
1. Choisissez un hébergeur qui supporte le *déploiement continu* (ex. [Netlify](/fr/guides/deploy/netlify/) ou [Vercel](/fr/guides/deploy/vercel/)) et importez votre dépôt Git en tant que nouveau projet / site.
```